### DISCUSSION TITLE
mini.starter removes buffer number lines

### DISCUSSION THREAD

**Discussion by ldelossa** [nvim-mini/mini.nvim#720] [Q&A] [answered]
My lua config sets line numbers on.

However, I want to use mini.starter with the example in the readme:
```
now(function()
  local starter = require('mini.starter')
	starter.setup({
		autoopen = true,
    evaluate_single = true,
    items = {
      starter.sections.builtin_actions(),
      starter.sections.recent_files(10, false),
      starter.sections.recent_files(10, true),
      starter.sections.sessions(5, true)
    },
    content_hooks = {
      starter.gen_hook.adding_bullet(),
      starter.gen_hook.indexing('all', { 'Builtin actions' }),
      starter.gen_hook.padding(3, 2),
    },
}) end)
```

What I'm finding tho is the starter page starts with `set nonumbers`. 
Is there a way to turn `set numbers` back on after the starter page is dismissed?

**Comment by echasnovski** [comment, marked as answer]
> My lua config sets line numbers on.

Yes, mine too. 'mini.starter' only disables number column for when the Starter buffer is shown in the window. Otherwise it should use global value of `number`.

Do you have any reproducible steps to see that `number` is not shown after picking some item in Starter buffer?

**Reply by ldelossa** [reply]
Ah sorry, you're right. I didn't copy-pasta `vim.opt.number = true` to my new mini.nvim config. My bad.
